browser/greaselion/greaselion_service_factory.cc

```diff
@@ -12,9 +12,11 @@
 #include "base/no_destructor.h"
 #include "base/path_service.h"
 #include "brave/browser/brave_browser_process.h"
+#include "brave/components/brave_rewards/common/pref_names.h"
 #include "brave/components/greaselion/browser/greaselion_service.h"
 #include "brave/components/greaselion/browser/greaselion_service_impl.h"
 #include "chrome/browser/extensions/extension_service.h"
+#include "chrome/browser/profiles/profile.h"
```

Comment:
Can we check if these need deps?

Reply:
thanks, updated `sources.gni`.

Frankly, I don't full understand that approach with "components" in brave/browser/ until chrome does the same.
If fact, we push all the deps into a one list and non on them can be build separately.